### ISSUE TITLE
val_acc vs. val_accuracy

### ISSUE THREAD

**Issue by andrewjmc** [jessieren/DeepVirFinder#12]
Running on Windows within Miniconda3 I get the warning below. Does val_acc need to be changed to val_accuracy in the early stopping rules?

`C:\Users\...\miniconda3\envs\dvf\lib\site-packages\keras\callbacks\callbacks.py:846: RuntimeWarning: Early stopping conditioned on metric `val_acc` which is not available. Available metrics are: val_loss,val_accuracy,loss,accuracy
  (self.monitor, ','.join(list(logs.keys()))), RuntimeWarning`

**Comment by jessieren** [comment]
Hi Andrewjmc, 

Thanks for reporting the issue. Keras has been updated so it is possible that the semantics changed. Hope it worked after you changed it to val_accuracy.
